src/Microsoft.Health.Fhir.Core/Features/Operations/DataConvert/IDataConvertEngine.cs

```diff
@@ -0,0 +1,16 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.Fhir.Core.Messages.DataConvert;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.DataConvert
+{
+    public interface IDataConvertEngine
```

Comment:
Did we consider `DataConversion` instead of `DataConvert`?

Reply:
Yeah, I also prefer DataConversion. Should I change all DataConvert to DataConversion in the class/variable naming? Seems all previous operations are in a verb form (validate, reindex).

Reply:
Per our offline discussion, have changed to convert-data